cernopendata/modules/theme/assets/semantic-ui/js/glossary/jquery.zglossary.js

```diff
@@ -27,13 +27,13 @@ import jquery from "jquery/dist/jquery";
 				var skip = 0;
 
 				// Check the element is a text node
-				if (e.nodeType == 3) {
+				if (e.nodeType == 3 && e.textContent.trim().length) {
 
 					// Case insensistive matching option
 					if (options.ignorecase) {
-						var pos = e.data.toLowerCase().search(new RegExp('\\b'+patfmt.toLowerCase()+'\\b'));
+						var pos = e.data.toLowerCase().search(new RegExp('(?<!\\\\)\\b'+patfmt.toLowerCase()+'\\b'));
```

Comment:
AFIAK we have no other option than this ugly [escaping](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping) since we have to use a string literal for the regexp (as a regular expression literal it looks like [`'(?<!\\)\btau\b'`](https://regex101.com/r/R5mDa3/1)).

Reply:
Thanks, seems to fix the several tau troubles I have tried :+1: 

The font size jumping is also behaving better. I guess we can accept the "similar size"

Reply:
> this ugly escaping

... BTW your sigh reminds me of the quote by Jamie Zawinski (see [history](http://regex.info/blog/2006-09-15/247)): _"Some people, when confronted with a problem, think “I know, I'll use regular expressions.”   Now they have two problems."_ :wink: I agree this is a forced move here...